class CreateUserError(Exception):
    pass


class UserAlreadyExistsError(Exception):
    pass
